spec/views/catalog/_index_list_default.html.erb_spec.rb

```diff
@@ -27,7 +28,8 @@
 
   it 'displays desired fields' do
     expect(rendered)
-      .to include('<span class="attribute-label h4">Keyword:</span>', 'keyword',
+      .to include('<span class="attribute-label h4">Creator:</span>', 'creator',
+                  '<span class="attribute-label h4">Keyword:</span>', 'keyword',
                   '<span class="attribute-label h4">Resource Type:</span>', 'resource_type')
```

Comment:
it feels fragile to include the 'h4' class.  Could we re-write this to look for nodes with class==attribute-label and the associated values?  I.E. what if someone decides all of these should be h5 in the future or <li> elements instead of <span>, etc.  I do think it's fair to assume that class="attribute-label" should be stable.

Reply:
I like the idea of writing a shared example for `is_expected.to have_index_display_field` or similar that is more aware of the class.

This test is refactored slightly from the Hyrax specs.

I'm okay holding this PR in lieu of that change, or merging it and taking on a refactor as we work on other fields.

Reply:
Let's merge as it is to avoid blocking future work.  I like the idea of a matcher in the future.